src/SQLCache.php

```diff
@@ -317,7 +319,7 @@ public function increment($key, $step = 1, $ttl = null)
             $next = ($current ?? 0) + $step;
             if ($current !== null) {
                 $stmt = $this->getStatement(__METHOD__, self::UPDATE_INCREMENT_SQL);
-                $result = $stmt->execute([$next, $key]) && $stmt->rowCount() === 1;
+                $result = $stmt->execute([serialize($next), $key]) && $stmt->rowCount() === 1;
```

Comment:
I think this makes it backwards-incompatible? The serialize call will make this break incremented/decremented values.

Reply:
They're already serialized, actually. The get/set do it normally but since I'm bypassing them for the update case, I need to do it here too. Not serializing is why the tests failed in the first commit.